internal/cmd/cmd.go

```diff
@@ -78,7 +83,18 @@ func defaultAPIClient() (*api.Client, error) {
 		return nil, err
 	}
 
-	return apiClient(config.Host, config.AccessKey, httpTransportForHostConfig(config)), nil
+	server := config.Host
+	accessKey := config.AccessKey
+
+	if envAccessKey, isSet := os.LookupEnv("INFRA_ACCESS_KEY"); isSet {
+		accessKey = envAccessKey
+	}
+
+	if envServer, ok := os.LookupEnv("INFRA_SERVER"); ok {
+		server = envServer
+	}
```

Comment:
To be clear, this functions as an override correct? So it can be any combination of server from config or env var and access key from config or env var?